app/views/steps/submission/shared/_money_answer.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
```

Comment:
I have used the existing answer component pattern here for consistency. I prefer the pattern I've used for GroupedList and Savings, but updating existing components is beyond the scope of this workset.